index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">    
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>    
+    <nav class="nav">  
+        <div class="logo"> Edahi </div>
+        <a href="#" class="hamburger-nav">
+            <img  src="./images/Hamburger.png"> 
+        </a>        
+
+    </nav>
+    <section class="presentation">
+        <header class="text-1">
+            Hey There.
+            <div>I’m Edahi</div>
+        </header>
+
+        <div class="text-2">
```

Comment:
- [ ] Please as requested by the previous reviewer, make sure to wrap your `I'm a Software Developer` text inside an `h2` instead of a `div`.